front-page.php

```diff
@@ -0,0 +1,31 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+
+  <section id="player">
+    <?php // get_template_part('partials/player'); ?>
+  </section>
+
+  <section id="page-programacion" class="page-overlay">
+    <?php get_template_part('partials/programacion'); ?>
+  </section>
+
+<?php
+  $page_sobre = get_page_by_path('sobre');
+
+  if ($page_sobre != null) {
+?>
+  <section id="page-sobre" class="page-overlay">
+    <?php echo apply_filters('the_content', $page_sobre->post_content); ?>
```

Comment:
Would it be nicer to setup post data here so you can just use `the_content()` and maybe in the future use other template tags? I'm not sure what is best practice